src/components/editor/diacritics/DiacriticsSelection.jsx

```diff
@@ -36,7 +36,19 @@ const DiacriticsSelection = () => {
     event.preventDefault()
   }
 
-  return (<div id="diacritics-selection" className="sticky-top bg-white" tabIndex="0">
+  const keyPressHandler = (event) => {
+    switch (event.which) {
+      case 8:
+        event.preventDefault() // backspace should not be passed to the browser as it causes the page to go back
+        break
+      case 27:
+        dispatch(hideDiacritics()) // esc should close the panel, all other keys can pass through
+        break
+      // no default
+    }
+  }
+
+  return (<div id="diacritics-selection" role="presentation" onKeyDown={keyPressHandler} className="sticky-top bg-white" tabIndex="0">
```

Comment:
Shouldn't this be `role="dialog"`?

Reply:
Yeah, I had the same thought but it made the linter mad:

``` 51:11  error  Non-interactive elements should not be assigned mouse or keyboard event listeners  jsx-a11y/no-noninteractive-element-interactions```

But we really want a keyboard listener on this element because it is only way we could find to trap the backspace key.

Reply:
Why is it on the modal in particular and not on the the whole page?  I'm seeing the backspace navigate back to the previous page regardless of whether the diacritics dialog is open.

Reply:
I was just responding to this particular ticket - not sure if that backspace key going back in the browser is desired in other contexts, so didn't want to unilaterally disable it everywhere.  The esc key was just a gratuitous addition that made sense to me.

Reply:
@jermnelson @astridu ?

Reply:
Doing some testing, the backspace issue exists only for Firefox and not Chrome (although you can download an [extension](https://chrome.google.com/webstore/detail/enable-backspace/ddijblnjiploanefjnbpimgbfjkpifia?hl=en#:~:text=Enable%20Backspace%20extension%20enables%20you,this%20functionality%20to%20your%20browser.) to restore this functionality). 

For example (all using Firefox):
*  Open  the editor with the four tabs **Resource Templates**, **Search**, **Load RDF**, and **Exports**. Click on any of **Search**, **Load RDF**, and **Exports** tabs, click backspace returns you to the **Resource Templates** tab.
*  Open a resource template from the **Resource Templates** and after the template is loaded into the **Editor** tab, hitting backspace returns you to the **Resource Templates** tab. However, we have the following wrinkles (I used the [ld4p:RT:bf2:Monograph:Work:Un-nested](https://development.sinopia.io/editor/ld4p:RT:bf2:Monograph:Work:Un-nested) template on development):
   *  Adding any text to a top-level property (used **Date of Work**) and hitting enter prevents the backspace from going back to the **Resource Templates** tab.
   *  Starting from newly loaded page with the Work template, adding text to a nested property (used **Title Information > Work Title > Preferred Title of Work**) and hitting enter to save the value but hitting the backspace goes back to the **Resource Templates** tab.

Working on this branch, when opening a template and opening the Diacritics panel but not selecting any vocabulary, hitting the backspace returns you to the **Resource Templates** tab. If you select any of the vocabularies and display the character map, the fix works. 

Given this behavior, I second @jcoyne suggestion to see if we can add this to the whole page to see if we can prevent the different examples above where hitting backspace returns to the previous page.

Reply:
Just one thing to note, this behavior is not totally unexpected.  It turns out that Firefox and Chrome have different shortcut keys for going back a page.  Firefox uses delete, and Chrome removed that shortcut a couple years ago and uses a different key combination now instead.  The examples you site above seem to indicate that when the focus is in a text entry field, Firefox makes the delete key behave as expected (delete a key in the field being edited), and when the focus is elsewhere, it uses the default Firefox behavior of going back a page.

Just highlighting this, as I don't think this is a bug necessarily - it's acting the way the browser is intended to act.  I really like the idea of disabling for diacritics panel usage, because that is a mode where the user is entering text by clicking buttons (so they are in a sort of "text entry" mindset).  The other contexts are less clear.  Still experimenting with whole screen blocking if possible.